admin_guide/sdn_troubleshooting.adoc

```diff
@@ -0,0 +1,547 @@
+= Troubleshooting OpenShift SDN
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As described in the
+link:/architecture/additional_concepts/sdn.html[SDN documentation]
+there are multiple layers of interfaces that are created to correctly
+pass the traffic from one container to another.  In order to debug
+connectivity issues, you have to test the different layers of the
+stack to work out where the problem arises.  This guide will help you
+dig down through the layers to identify the problem and how to fix it.
+
+Part of the problem is that OpenShift can be set up many ways, and the
+networking can be wrong in a few different places.  So this document
+will work through some scenarios that, hopefully, will cover the
+majority of cases.  If your problem is not covered, the tools and
+concepts that are introduced should help guide debugging efforts.
+
+
+== Nomenclature
+
+Cluster:: The set of machines in the cluster.  _i.e._ the Master Nodes
+and the Client Nodes.
```

Comment:
This topic would be the only place where we refer to masters as "master nodes" and nodes as "client nodes". Do you feel it's important to make these distinctions in this topic? Or throughout, could it just be kept to "masters" and "nodes" when referring to OpenShift cluster members.